packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -891,11 +890,25 @@ export class PaginatedMessage {
 	protected setUpCollector(channel: TextBasedChannel, targetUser: User): void {
 		if (this.pages.length > 1) {
 			this.collector = new InteractionCollector<MessageComponentInteraction>(targetUser.client, {
-				filter: (interaction) => interaction.isMessageComponent() && this.actions.has(interaction.customId),
+				filter: (interaction) =>
+					!isNullish(this.response) &&
+					interaction.message.id === this.response?.id &&
```

Comment:
Theoretically, these lines are no longer needed because of the message filter below but I don't feel confident enough to remove them. Thoughts Vladdy/Kyra?

I did a quick test with them disabled and it does seem to work fine still.

Reply:
I have no idea..